aiohttp/connector.py

```diff
@@ -573,6 +574,13 @@ def _resolve_host(self, host, port):
                 host, port, family=self._family)
             return res
 
+    def _host_is_ip(self, host):
+        try:
+            ipaddress.ip_address(host)
+            return True
+        except ValueError:
+            return False
```

Comment:
Anything else to suggest? The ipaddress module does not have a way to verify the format while avoiding the exception. Custom-made validation does not seem like a better route.